server.js

```diff
@@ -1,2 +1,81 @@
-require('dotenv').config()
-const express = require('express')
\ No newline at end of file
+//required dependencies
+const mongoose = require("mongoose");
+const express = require("express");
+const app = express();
+const path = require("path");
+const { response } = require("express");
+
+//local variables
+const staticDir = path.resolve('./client/public')
+const port = process.env.PORT || 5000;
+
+//database setup
+mongoose.connect("mongodb://localhost:27017/til", {
+    useNewUrlParser: true,
+    useUnifiedTopology: true
+});
+    
+const tilDB = mongoose.connection;
+tilDB.on("error", console.error.bind(console, "connection error:"))
+
+const FormSchema = new mongoose.Schema({
+    title: String,
+    author: String,
+    topic: String,
+    content: String,
+    date: Date
+})
+
+const FormModel = new mongoose.model("Form", FormSchema);
+
+//Server setup
+
+app.use(express.static(staticDir))
+app.use(express.urlencoded({ extended: true }));
+
+//post requests will create a new post unless duplicate title
+app.post("/facts", async (req, res) => {
+    let formInfo = req.body;
+    let formObj = formModel.findOne({ title: formInfo.title });
```

Comment:
Initially when I submit a new form the whole program errors out, and the terminal shows an error that `formModel is not defined`. It looks like this should be `FormModel` like you declared on line 29? However once making this change the program hits the error block on line 48. It looks like you are on the right track.. but will need to do some more debugging to get things up and running 100%